compiler/rustc_borrowck/src/type_check/liveness/trace.rs

```diff
@@ -36,7 +37,7 @@ use crate::type_check::{NormalizeLocation, TypeChecker};
 /// DROP-LIVE set are to the liveness sets for regions found in the
 /// `dropck_outlives` result of the variable's type (in particular,
 /// this respects `#[may_dangle]` annotations).
-pub(super) fn trace<'mir, 'tcx>(
+pub(super) fn trace2<'mir, 'tcx>(
```

Comment:
Question: presumably this collides with tracing::trace, which is unfortunate. Could we disambiguate this with

- tracing::trace, and
- dataflow::trace

where needed? Or is that too confusing?

Reply:
Oh, I accidentally left that behind. I will undo that change.